configs/components/echam/echam.yaml

```diff
@@ -836,6 +859,24 @@ grids:
                 oasis_grid_type: "D"
                 number_of_overlapping_points: 0 # oasis P-value
 
+choose_lgfw:
+        True:
+                add_input_sources:
+                        gfw_atmo: "${general.experiment_couple_dir}/latest_gfw_atmo.nc"
+                add_input_files:
+                        gfw_atmo: gfw_atmo
+                add_input_in_work:
+                        gfw_atmo: "gfw_atmo.nc"
```

Comment:
this can always be in the general `input_sources`, to reduce the amount of verbosity. Only `input_files` decide which `input_files` from the available sources go into the simulation.